src/components/dialogs/dialogUtils.js

```diff
@@ -33,6 +33,11 @@ export const useStyles = makeStyles((theme) => ({
         fontSize: 'small',
         marginTop: theme.spacing(1),
     },
+    emptyListError: {
+        color: theme.palette.error.main,
+        textDecorationLine: 'overline',
```

Comment:
Remove overline maybe.
And the font size is too big. It should be like the other error messages

Reply:
the other errors messages are default style of input control , ill remove the overline